src/cli.js

```diff
@@ -17,10 +17,13 @@ const chalk = require('chalk');
 
 const defaultConfig = require('./config/adobeioruntime-node10.js');
 
+require('dotenv').config();
+
 class CLI {
   constructor() {
     this._yargs = yargs()
       .pkgConf('wsk')
+      .env('HLX')
```

Comment:
This might clash with helix-cli, but probably won't matter as helix-cli would not use the hedy cli

Reply:
> This might clash with helix-cli, but probably won't matter as helix-cli would not use the hedy cli

but it helps putting the params/secrets in an `.env`. which helps with deployment in CI.
so far, the openwhisk credentials were read from the `.wskprops` e.g. in circleci.